modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -19,16 +19,26 @@ If you miss this step, the second boot will fail.
 For example, to configure DHCP networking, identify `ip=dhcp`
 or set static networking when you add parameters to the kernel command line.
 
-. Generate the thumbprint. Install the clevis package, it is not already
-installed, and generate a thumbprint
-from the Tang server. Replace the value of `url` with the Tang server URL:
+. Generate the thumbprint.
+..Install the clevis package, if it is not already installed:
 +
+[source,terminal]
 ----
 $ sudo yum install clevis -y
+----
+
+.. Generate a thumbprint from the Tang server. Replace the value of `url` with the Tang server URL:
++
+[source,terminal]
+----
 $ echo nifty random wordwords \
      | clevis-encrypt-tang \
        '{"url":"https://tang.example.org"}'
-
+----
++
+.Example output
```

Comment:
I understand why input and output are being separated in our new style. However, the challenge here is that the "Example output" is a mix of input and output. So the "y" after the words "Do you wish to trust..." is actually something the user types.